### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by sebastianhelbig** [miracle2k/tarsnapper#42]
- fresh (previously nuked) tarsnapper "instance"
- first backup completes
- on second run:

```
Creating backup www.example.com: www.example.com-20170316-070936
Executing: tarsnap -c -f www.example.com-20170316-070936 /srv/www.example.com
tarsnap: Removing leading '/' from member names^M
Something went wrong with backup job: 'www.example.com'
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/tarsnapper-0.4.0-py2.7.egg/tarsnapper/script.py", line 381, in run
    self.backend.make(job)
  File "/usr/lib64/python2.7/site-packages/tarsnapper-0.4.0-py2.7.egg/tarsnapper/script.py", line 223, in make
    self.call(*args)
  File "/usr/lib64/python2.7/site-packages/tarsnapper-0.4.0-py2.7.egg/tarsnapper/script.py", line 65, in call
    return self._exec_tarsnap(call_with)
  File "/usr/lib64/python2.7/site-packages/tarsnapper-0.4.0-py2.7.egg/tarsnapper/script.py", line 78, in _exec_tarsnap
    has_prompt = (child.expect([u'Please enter passphrase for keyfile .*?:', pexpect.EOF]) == 0)
  File "/usr/lib64/python2.7/site-packages/pexpect-4.2.1-py2.7.egg/pexpect/spawnbase.py", line 321, in expect
    timeout, searchwindowsize, async)
  File "/usr/lib64/python2.7/site-packages/pexpect-4.2.1-py2.7.egg/pexpect/spawnbase.py", line 345, in expect_list
    return exp.expect_loop(timeout)
  File "/usr/lib64/python2.7/site-packages/pexpect-4.2.1-py2.7.egg/pexpect/expect.py", line 99, in expect_loop
    incoming = spawn.read_nonblocking(spawn.maxread, timeout)
  File "/usr/lib64/python2.7/site-packages/pexpect-4.2.1-py2.7.egg/pexpect/pty_spawn.py", line 465, in read_nonblocking
    return super(spawn, self).read_nonblocking(size)
  File "/usr/lib64/python2.7/site-packages/pexpect-4.2.1-py2.7.egg/pexpect/spawnbase.py", line 163, in read_nonblocking
    self._log(s, 'read')
  File "/usr/lib64/python2.7/site-packages/pexpect-4.2.1-py2.7.egg/pexpect/spawnbase.py", line 121, in _log
    self.logfile.write(s)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xdc' in position 66: ordinal not in range(128)
```

**Comment by miracle2k** [comment]
It seems that the recently added Python 3 support is a bit broken; you can use an older version. 

If someone could contribute fixes in this regard, I would appreciate it.

**Comment by ktrhn** [comment]
I can confirm that downgrading to tarsnapper 0.3.0 resolved the problem.

**Comment by ralish** [comment]
The problem here is the setting/overriding of the `LANG` environment variable before invoking Tarsnap:
https://github.com/miracle2k/tarsnapper/blob/master/tarsnapper/script.py#L70

I'm not quite sure what the intent of this is, but the result isn't that Tarsnap output is in English. Rather, the character encoding is forced to the C locale, which effectively limits the output to ASCII characters. If any output from Tarsnap can't be represented in ASCII, we hit a `UnicodeEncodeError` exception.

What's occurring in my and @sebastianhelbig's cases is Tarsnap is processing files with Unicode characters in their names, but because we've forced the encoding to C locale, these names can't be represented.

I've submitted PR #49 to remove this variable outright as I'm not sure what the point is of setting it. If the system is configured to be Unicode aware, we should respect that, as we can reasonably expect to encounter files with UTF-8 names. If the system isn't Unicode aware, the locale environment we inherit should already reflect that.
